app/src/test/java/shop/woowasap/accept/support/valid/ShopValidator.java

```diff
@@ -28,13 +28,12 @@ public static void assertProductsFound(ExtractableResponse<Response> result,
         HttpValidator.assertOk(result);
 
         ProductsResponse resultResponse = result.as(ProductsResponse.class);
-        assertProducts(resultResponse, expected);
+        assertThat(resultResponse).usingRecursiveComparison().isEqualTo(expected);
```

Comment:
질문 : 여기 id값 비교 제외는 안해줘도 되나요??

```suggestion
        assertThat(resultResponse).usingRecursiveComparison().isEqualTo(expected);
```

Reply:
id 값 비교는 제외해야합니다. 추가할게요!